<ui version="4.0" >
 <author>Konrad Wilhelm Kleine</author>
 <class>GuiPrinterDriverToXmlBase</class>
 <widget class="QWidget" name="GuiPrinterDriverToXmlBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CUPS Driver 2 XML</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="textLabel3" >
     <property name="text" >
      <string>This program collects information from all PPD files under the specified directory. 
After that you can save this information in a XML file for use with the printer setup utility.</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_build" >
     <property name="title" >
      <string>Directories</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="textLabel2" >
          <property name="text" >
           <string>CUPS model dir</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editCupsModelDir" >
          <property name="text" >
           <string>/usr/share/cups/model/</string>
          </property>
          <property name="readOnly" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonBrowseCupsDir" >
          <property name="text" >
           <string>Br&amp;owse</string>
          </property>
          <property name="shortcut" >
           <string>Alt+O</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonBuild" >
          <property name="text" >
           <string>Bui&amp;ld</string>
          </property>
          <property name="shortcut" >
           <string>Alt+L</string>
          </property>
          <property name="default" >
           <bool>true</bool>
          </property>
          <property name="flat" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_progress" >
     <property name="title" >
      <string>Progress</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="textLabel1_3" >
          <property name="text" >
           <string>Manufacturers:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumberManufacturers" >
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="segmentStyle" >
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_2" >
          <property name="text" >
           <string>Models:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumberModels" >
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="segmentStyle" >
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Drivers:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumberDrivers" >
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="segmentStyle" >
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAbort" >
        <property name="text" >
         <string>Abor&amp;t 
build process</string>
        </property>
        <property name="shortcut" >
         <string>Alt+T</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_log" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Log</string>
     </property>
     <layout class="QGridLayout" >
      <item rowspan="2" row="0" column="0" >
       <widget class="QTextEdit" name="textEditLog" >
        <property name="undoRedoEnabled" >
         <bool>false</bool>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="buttonCopyToClipboard" >
        <property name="text" >
         <string>Cop&amp;y log 
to clipboard</string>
        </property>
        <property name="shortcut" >
         <string>Alt+Y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="buttonClearLog" >
        <property name="text" >
         <string>Clear Log</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_save" >
     <property name="title" >
      <string>Save to XML file</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="textLabel1_4" >
          <property name="text" >
           <string>Path to XML file</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editSavePath" >
          <property name="text" >
           <string>~/Desktop/cupsdb.xml</string>
          </property>
          <property name="readOnly" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonBrowseSavePath" >
          <property name="text" >
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSave" >
          <property name="text" >
           <string>Sa&amp;ve</string>
          </property>
          <property name="shortcut" >
           <string>Alt+V</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer name="spacer6" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBack" >
       <property name="text" >
        <string>Bac&amp;k</string>
       </property>
       <property name="shortcut" >
        <string>Alt+K</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>textEditLog</tabstop>
  <tabstop>buttonCopyToClipboard</tabstop>
  <tabstop>buttonClearLog</tabstop>
  <tabstop>editCupsModelDir</tabstop>
  <tabstop>buttonBrowseCupsDir</tabstop>
  <tabstop>buttonBuild</tabstop>
  <tabstop>buttonAbort</tabstop>
  <tabstop>editSavePath</tabstop>
  <tabstop>buttonBrowseSavePath</tabstop>
  <tabstop>buttonSave</tabstop>
  <tabstop>buttonBack</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
